src/main/java/com/ocelotslovebirds/birdhaus/blocks/BirdhouseBlockEntity.java

```diff
@@ -180,8 +184,25 @@ private void removeBirdHouseGoalToSurroundingParrots() {
     private void handleSeedsForTick() {
         // Tick the birdhouse to check it needs to consume a seed
         if (tickerForSeedConsumption.tick()) {
+            ItemStack seeds = itemHandler.extractItem(0, 1, false);
+
+            if (seeds.is(Tags.Items.SEEDS_WHEAT)){
+                birdColor = 3;
+            }
+            else if (seeds.is(Tags.Items.SEEDS_BEETROOT)){
+                birdColor = 1;
+            }
+            else if (seeds.is(Tags.Items.SEEDS_MELON)){
+                birdColor = 2;
+            }
+            else if (seeds.is(Tags.Items.SEEDS_PUMPKIN)){
+                birdColor = 4;
+            }
+            else {
+                birdColor = 0;
+            }
```

Comment:
I think it would be nice to mention what each of the digits correspond to

Reply:
Yeah I will add comments to that, just haven't gotten around to it. I'm also thinking of trying to refactor into a switch statement to make it cleaner

Reply:
im not sure if a switch statement is good here tbh

Reply:
Yeah possibly

Reply:
done